test/tests/adapters/mongoose.test.js

```diff
@@ -685,6 +728,23 @@ describe('adapters > schema-adapter', () => {
       expect(result).toHaveProperty('fields');
       expect(result.fields[0]).toHaveProperty('reference', 'Bar._id');
     });
+
+    it('should not set the ref attribute if it does not exist', async () => {
+      expect.assertions(3);
+      const warnSpy = jest.spyOn(Interface.logger, 'warn');
+      const schema = new mongoose.Schema({
+        foo: { type: String, ref: 'Bar' },
+      });
+      const model = mongoose.model('Foo', schema);
+      const result = await createSchemaAdapter(model, {
+        Mongoose: mongoose,
+        connections: { mongoose },
+      });
+
+      expect(warnSpy).toHaveBeenCalledWith('Cannot find the reference "Bar" on the model "Foo".');
+      expect(result).toHaveProperty('fields');
+      expect(result.fields[0]).not.toHaveProperty('reference', 'Bar._id');
```

Comment:
should this test be `      expect(result.fields[0]).not.toHaveProperty('reference');`

as in the previous version, if I understand the property was `null._id`